### PR TITLE
Add documentation for messages

### PR DESCRIPTION
**Pull request by R4356th**
Translating without any context is really hard.

### CODE REVIEW

File: i18n/qqq.json [hunk 1]
```diff
@@ -13,7 +14,33 @@
 	},
 	"action-managewiki": "Button to manage the settings of a wiki.",
 	"apihelp-query+wikiconfig-summary": "{{doc-apihelp-summary|wikiconfig}}",
-	"logentry-managewiki-settings": "Used as log message.",
+	"apihelp-query+wikiconfig-example-1": "{{doc-apihelp-example|wikiconfig}}",
+        "apihelp-query+wikiconfig-param-prop": "{{doc-apihelp-param|wikiconfig|prop}}",
```
Comment (R4356th) [inline i18n/qqq.json:18]: Not sure why the indentation is showing up like this. I am sure I did it correctly.

Comment (Universal-Omega) [reply i18n/qqq.json:18]: You have spaces here not tabs. Should be tabs

Comment (R4356th) [reply i18n/qqq.json:18]: Hmm... Let me try...

Comment (R4356th) [reply i18n/qqq.json:18]: GitHub does not like it. I will test this once I add more messages.

**Pull request merged**